pkg/pillar/types/global.go

```diff
@@ -704,7 +704,7 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddIntItem(Dom0MinDiskUsagePercent, 20, 20, 0xFFFFFFFF)
 
 	// Add Bool Items
-	configItemSpecMap.AddBoolItem(UsbAccess, true)
+	configItemSpecMap.AddBoolItem(UsbAccess, true) // Controller likely default to false
```

Comment:
Should this be false? This is supposed to be the default value if the config item is not specified in the Cloud conifg?